### DISCUSSION TITLE
Will a wiki be implemented? I don't see any difference with or w/o the extension

### DISCUSSION THREAD

**Discussion by lundeen-bryan** [znck/grammarly#239] [Q&A] [answered]
As the title states, I don't see a difference, I see A LOT of settings but I have no idea what they do. How do I link the extension with my Grammarly account so that it will help me? How do I use it in a markdown file? Is it only for a README.md file or any markdown file? 

I really appreciate the idea of this extension but I don't know how to use it.

**Comment by lundeen-bryan** [comment, marked as answer]
So sorry, I figured it out. Have to go to the settings and looked for the section called "Grammarly: Patters."
Then to use Grammarly on all markdown files, regardless of being named "README.md" or not, it will work. 
Click "Add Item" > then type the same text in the quote here: "**/*.md" then press the OK button. 
Then you will notice it works on any markdown file.

**Reply by znck** [reply]
#240 Will add a command, `Grammarly: Check text`, to add files to `Grammarly: Selectors` preferences section.
